vapid.js

```diff
@@ -32,10 +48,10 @@ var vapid = {
            .then(keys => {
               this._private_key = keys.privateKey;
               this._public_key = keys.publicKey;
-              console.info("VAPID Keys defined.");
+              console.info(this.errs.enus.OK_VAPID_KEYS);
```

Comment:
Possibly. I was considering that these are strings that users could wind up being read by users. There are fun tricks one can do with getting user agent languages and have these autoswitch.